payload/reggae/dub/interop/dublib.d

```diff
@@ -162,67 +167,41 @@ public void fetchDubDeps(in string projectPath) @trusted {
 }
 
 
-/// Normally ~/.dub
-string userPackagesPath() @safe {
-    import reggae.path: buildPath;
-    import std.process: environment;
-    import std.path: isAbsolute;
-    import std.file: getcwd;
-
-    version(Windows) {
-        immutable appDataDir = environment.get("APPDATA");
-        const path = buildPath(environment.get("LOCALAPPDATA", appDataDir), "dub");
-    } else version(Posix) {
-        string path = buildPath(environment.get("HOME"), ".dub/");
-        if(!path.isAbsolute)
-            path = buildPath(getcwd(), path);
-    } else
-          static assert(false, "Unknown system");
+public string dubPackagePath(in string packageName, in string version_) @trusted {
+    import dub.dub: Dub;
+    import dub.dependency: Version;
+    import dub.packagemanager: PlacementLocation;
+    import std.exception: enforce;
+    import std.conv: text;
 
-    return path;
-}
-
-string systemPackagesPath() @safe {
-    import reggae.path: buildPath;
-    import std.process: environment;
+    auto dub = new Dub(".");
+    auto pkg = dub.packageManager.getPackage("dub", Version("1.34.0"), PlacementLocation.user);
```

Comment:
Why do you force the `PlacementLocation` though ?

Reply:
More importantly, both `packageName` and `version_` params are unused here, using hardcoded values instead.

Reply:
And annoyingly, there's a different function for git+https selections (`packageManager.loadSCMPackage()`).

Reply:
Yeah I need to fix that. Having the `PackageManager` fetch things doesn't seem like the right call.

Reply:
> Why do you force the PlacementLocation though ?

No idea, I used the first thing that worked. I've since decided that this is too complicated and slow anyway and that instead I'm going to vendor dub and use a path-based dependency for the code instead.

Reply:
> More importantly, both `packageName` and `version_` params are unused here, using hardcoded values instead.

This is what happens when one doesn't have others review their PRs... oops.